src/App/App.js

```diff
@@ -65,26 +64,22 @@ function App() {
       });
   }
 
-  const increaseVote = (id) => {
+  const handleVote = (id, direction) => {
     const updatedMovies = movies.map((movie) => {
       if (movie.id !== id) {
         return movie;
       }
-
-      updateVote(movie.id, "up");
-      return { ...movie, vote_count: movie.vote_count + 1 };
-    });
-    setMovies(updatedMovies);
-  };
-
-  const decreaseVote = (id) => {
-    const updatedMovies = movies.map((movie) => {
-      if (movie.id !== id) {
-        return movie;
+      updateVote(movie.id, direction);
+      let updatedVoteCount = movie.vote_count;
```

Comment:
Way to clean up the code, love it when we can merge many functions into something slim and efficient .